src/components/homePage/Contributors.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div id="contributors">
+    <h1>Created with love in 🇨🇦 by</h1>
+    <div class="gallery">
+      <div class="parent" v-for="(member,index) in members" :key="member.name">
+        <a @click="open(index)" style="text-decoration: none;">
+          <div class="dialog" v-if="popUp===index">
+            <div class="name">{{member.name}}</div>
+            <div class="emoji">{{member.emoji}}</div>
+            <div class="position">{{member.position}}</div>
+            <img class="rectangle" src="../../assets/Rectangle.svg" />
+          </div>
+          <div v-if="'image' in member">
+            <img class="image" :src="require(`@/assets/contributors/${member.image}`)" />
+            <!-- <img class="image" :src="require(`@/assets/contributors/${member.image}`)" /> -->
+          </div>
+          <div v-else>
+            <img class="image" src="../../assets/contributors/anon.png" />
+          </div>
+        </a>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import Members from "../../content/contributors.json";
+export default {
+  name: "Contributors",
+  data() {
+    return { members: Members, popUp: -1 };
+  },
+  methods: {
+    open: function(popped) {
+      this.popUp = popped;
+      console.log(this.popUp);
+    }
+  }
+};
+</script>
+
+<style scoped>
+#contributors {
+  color: black;
+  margin-top: 5rem;
+}
+.gallery {
+  display: flex;
+  flex-flow: row wrap;
+  margin-top: 2rem;
+}
+.parent {
+  position: relative;
+}
+.image {
+  max-height: 5rem;
+  margin-right: 3.5rem;
+  margin-bottom: 1rem;
```

Comment:
I'd encourage you to use `display: grid;` for this. It's definitely the right thing for the job - although flex works in this case, it does cause some sad moments

![image](https://user-images.githubusercontent.com/41529410/112080413-5555c780-8b58-11eb-9598-d1b22270a6db.png)
![image](https://user-images.githubusercontent.com/41529410/112080429-5e469900-8b58-11eb-9f1b-9c195e4d1da8.png)

Also, it'd be a great learning experience to use grid! We can call and I'll show you how to set it up. Or if you want, I can take this one change off our hands for you :)